.github/workflows/screener-build.yml

```diff
@@ -83,6 +92,33 @@ jobs:
         with:
           fetch-depth: 0
 
+      - uses: actions/setup-node@v3
```

Comment:
Can we create a reusable workflow to set env variables to avoid duplicating these step in all jobs ?

Reply:
I will create an issue for that

Reply:
looks like bad rebase, this is duplicate of line 92

<img width="1098" alt="image" src="https://user-images.githubusercontent.com/1223799/182865959-909361bb-2369-43c1-a05a-c3122c0b318f.png">